expressEngine/cms/product/success.js

```diff
@@ -1,4 +1,7 @@
 export default {
   userRegistered: 'You have registered successfully',
   productRegistered: 'Product registered successfully',
+  productDeleted: 'Successfully deleted Product',
+  productAlreadyDeleted: 'Product already deleted',
+  productNotFound: 'ProductNotFound',
```

Comment:
@DPDeepak replace 'ProductNotFound' with 'Product not found'.

Reply:
done